libtcmu_log.c

```diff
@@ -247,8 +268,7 @@ static bool log_buf_not_empty_output(struct log_buf *logbuf)
 	if (rb_is_empty(logbuf)) {
 		pthread_mutex_unlock(&logbuf->lock);
 		return false;
-	}
-
+	} 
 	tail = logbuf->tail;
```

Comment:
Just leave the newline between the chunks of code. No point in changing it in this patch.

In general if there is a coding style you want to change, do it in a patch that changes only the style.